docs/custom-matchers/writing-custom-matcher.md

```diff
@@ -15,8 +16,9 @@ const endsWithMatcher = pattern => {
       const wrapRegExp = new RegExp(`(.*)\.${pattern}$`); // In production application pattern should definitely be escaped!
 
       return {
-        unwrap: pattern,
-        wrap: action.type.match(wrapRegExp)[1],
+        id: `ends-with${pattern}`,
```

Comment:
Which reminds me that we should probably prefix all the built-in matchers with something like `@@redux-saga`